### PR TITLE
Add togglable Kafka/ActiveMQ Camel routes

### PR DESCRIPTION
**Pull request by geraldhumphries**
Eventd will push to the routes, and Alarmd is configured to pick up events from it

### CODE REVIEW

File: platform/events/daemon/src/main/java/org/opennms/netmgt/eventd/EventIpcManagerDefaultImpl.java [hunk 1]
```diff
@@ -129,6 +131,10 @@ public void rejectedExecution(Runnable r, ThreadPoolExecutor e) {
 
     private final MetricRegistry m_registry;
 
+    // magic?
```
Comment (j-white) [inline platform/events/daemon/src/main/java/org/opennms/netmgt/eventd/EventIpcManagerDefaultImpl.java:134]: Came is magic :)

**Comment by artnaseef** [comment]
The changes are deceptively simple.  Nice to see Camel slip in there smoothly.

File: platform/alarms/daemon/src/main/resources/OSGI-INF/blueprint/blueprint.xml [hunk 2]
```diff
@@ -70,14 +72,53 @@
...
+    </cm:property-placeholder>
+
+    <!-- Work around Camel using the wrong ActiveMQ component -->
+    <bean id="customamq" class="org.apache.camel.component.activemq.ActiveMQComponent">
+        <property name="brokerURL" value="${activeMqBrokerURL}"/>
+        <property name="usePooledConnection" value="false"/>
+    </bean>
+
+    <bean id="offsetFile" class="java.io.File">
+        <argument value="etc/alarm-kafka-offset.dat" />
```
Comment (artnaseef) [inline platform/alarms/daemon/src/main/resources/OSGI-INF/blueprint/blueprint.xml:99]: Let's have a configuration setting for this - it's OK to use a default.

File: platform/docker-it/src/test/resources/org/opennms/horizon/alarmsd-rest.feature [hunk 3]
```diff
@@ -30,7 +30,7 @@ Feature: OpenNMS Alarm Daemon Rest
     Given POST request body in resource "test-data/event001.xml"
     Then send POST request at path "/events"
     Then verify the response code 202 was returned
-    Then send GET request at path "/alarms/list" with retry timeout 20000
+    Then request alarm list with retry timeout 20000
```
Comment (artnaseef) [inline platform/docker-it/src/test/resources/org/opennms/horizon/alarmsd-rest.feature:33]: In the vien of "real obvious," how about `request non-empty alarm list with retry timeout 20000` (added "non-empty")

**Comment by artnaseef** [comment]
Manually merged into the main branch to resolve conflicts in features.xml.

**Pull request closed without merging**